src/spec-node/dockerCompose.ts

```diff
@@ -705,6 +705,7 @@ export function dockerComposeCLIConfig(params: Omit<PartialExecParameters, 'cmd'
 		return result || (result = (async () => {
 			let v2 = false;
 			let stdout: Buffer;
+			dockerCLICmd='docker-compose';
```

Comment:
Here `dockerCLICmd` should not be set to a docker compose command. Can we revert this?